model/textparse/nhcbparse.go

```diff
@@ -373,7 +372,14 @@ func (p *NHCBParser) processNHCB() bool {
 			p.hNHCB = nil
 			p.fhNHCB = fh
 		}
-		p.metricStringNHCB = p.tempLsetNHCB.Get(labels.MetricName) + strings.ReplaceAll(p.tempLsetNHCB.DropMetricName().String(), ", ", ",")
+
```

Comment:
Note to reviewers: the `ProtobufParser` parser doesn't add "{}" to the metric name if there are no labels. Instead of complicating `ProtobufParser` I've fixed this function. The metric names "metric{}" and "metric" are equivalent.